docs/get-started.md

```diff
@@ -123,3 +123,7 @@ composer require --dev bamarni/composer-bin-plugin
 composer bin phpinsights require nunomaduro/phpinsights
 ./vendor/bin/phpinsights
 ```
+
+## PHPStorm: link files from console
```

Comment:
Maybe this deserves other section, @olivernybroe what do you think?

Reply:
Hmm think you are right, just not sure where. 
What do you think about the idea to have a section for IDE's and a subsection for Phpstorm. This way we can also show how to configure phpinsights as an external tool in phpstorm under here. 
And later on add helping configuration for other IDE's?